app/assets/javascripts/fae/form/_filtering.js

```diff
@@ -152,7 +152,11 @@ Fae.form.filtering = {
    */
   sortingSetup: function() {
     this._addSortingClasses();
-    this._addSortingListeners();
+
+    if (!this._are_sorting_listeners_setup) {
+      this._addSortingListeners();
+      this._are_sorting_listeners_setup = true;
```

Comment:
Personal preference, but I'd elect to include this at the end of `_addSortingListeners()`. That way, if the variable is mutated again, it's always reset after this function is called. Otherwise you'll have to pray that whenever the function is invoked someone will always declare `this._are_sorting_listeners_setup` immediately afterward.